--- KLU/Makefile.orig	2009-11-07 18:37:57.000000000 +0100
+++ KLU/Makefile	2012-01-10 17:39:06.842722146 +0100
@@ -2,7 +2,7 @@
 
 default: demos
 
-include ../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 demos:
 	( cd Lib ; $(MAKE) )
--- KLU/Lib/Makefile.orig	2009-11-11 21:16:32.000000000 +0100
+++ KLU/Lib/Makefile	2012-01-10 17:41:46.846060837 +0100
@@ -2,7 +2,7 @@
 
 ccode: all
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 # for testing only:
 # TEST = -DTESTING
@@ -10,10 +10,10 @@
 C = $(CC) $(CFLAGS)
 
 INC = ../Include/klu.h ../Include/klu_internal.h ../Include/klu_version.h \
-    ../../SuiteSparse_config/SuiteSparse_config.h Makefile
+    Makefile
 
-I = -I../../AMD/Include -I../../COLAMD/Include -I../../BTF/Include \
-    -I../Include -I../../SuiteSparse_config
+I = -I/usr/include/amd -I/usr/include/colamd -I/usr/include/btf \
+    -I../Include
 
 all: library
 
--- KLU/Demo/Makefile.orig	2014-03-21 20:21:27.000000000 +0100
+++ KLU/Demo/Makefile	2014-10-03 16:36:20.732918931 +0200
@@ -3,23 +3,16 @@
 
 default: all
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
-CLIB = ../Lib/libklu.a  ../../BTF/Lib/libbtf.a \
-	../../AMD/Lib/libamd.a ../../COLAMD/Lib/libcolamd.a \
-        ../../SuiteSparse_config/libsuitesparseconfig.a
+CLIB = ../Lib/libklu.a -lbtf -lamd -lcolamd -lsuitesparseconfig
 
-CHOLMOD_LIB = ../../CHOLMOD/Lib/libcholmod.a
-CHOLMOD = $(CHOLMOD_LIB) $(LAPACK) $(BLAS) $(XERBLA)
+CHOLMOD = -lcholmod
 
-ifneq ($(GPU_CONFIG),)
-CHOLMOD += $(CUDART_LIB) $(CUBLAS_LIB) 
-endif
+I = -I/usr/include/amd -I/usr/include/colamd \
+	-I/usr/include/btf -I../Include -I/usr/include/cholmod
 
-I = -I../../SuiteSparse_config -I../../AMD/Include -I../../COLAMD/Include \
-	-I../../BTF/Include -I../Include -I../../CHOLMOD/Include
-
-all: $(CLIB) klu_simple $(CHOLMOD_LIB) kludemo kluldemo
+all: $(CLIB) klu_simple kludemo kluldemo
 	- ./klu_simple
 	- ./kludemo < ../Matrix/1c.mtx
 	- ./kludemo < ../Matrix/arrowc.mtx
@@ -37,24 +30,6 @@
 ../Lib/libklu.a:
 	( cd ../Lib ; $(MAKE) )
 
-../../BTF/Lib/libbtf.a:
-	( cd ../../BTF ; $(MAKE) library )
-
-../../AMD/Lib/libamd.a:
-	( cd ../../AMD ; $(MAKE) library )
-
-../../COLAMD/Lib/libcolamd.a:
-	( cd ../../COLAMD ; $(MAKE) library )
-
-../../CHOLMOD/Lib/libcholmod.a:
-	( cd ../../CHOLMOD ; $(MAKE) library )
-#	( cd ../../CAMD ; $(MAKE) )
-#	( cd ../../CCOLAMD ; $(MAKE) )
-#	( cd ../../metis-4.0 ; $(MAKE) )
-
-../../SuiteSparse_config/libsuitesparseconfig.a:
-	( cd ../../SuiteSparse_config ; $(MAKE) )
-
 purge: distclean
 
 distclean: clean
